cloudformation/security-hq-dev.template.yaml

```diff
@@ -0,0 +1,30 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Centralised security reporting for AWS
+
+Mappings:
+  Constants:
+    Stack:
+      Value: security
+    App:
+      Value: security-hq
+
+Resources:
+  SecurityHqIamDynamoTableDev:
+    Type: AWS::DynamoDB::Table
+    DeletionPolicy: Retain
+    Properties:
+      TableName: !Sub security-hq-iam-DEV
+      AttributeDefinitions:
+        - { AttributeName: id, AttributeType: S }
+      KeySchema:
+        - { AttributeName: id, KeyType: HASH }
+      ProvisionedThroughput: { ReadCapacityUnits: 5, WriteCapacityUnits: 5 }
+      Tags:
+        - Key: Stage
+          Value: DEV
+        - Key: Stack
+          Value:
+            Fn::FindInMap: [ Constants, Stack, Value ]
```

Comment:
As the template is small, I'd be tempted to inline these values and remove the complexity of `Mappings`.

Reply:
Yeah, agree, there's no real need to keep the approach from the PROD stack